assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego

```diff
@@ -0,0 +1,51 @@
+package Cx
+
+import data.generic.common as common_lib
+
+# for deprecated version (before AzureRM 3.0)
+CxPolicy[result] {
+	resource := input.document[i].resource.azurerm_app_service[name]
+	php_version := resource.site_config.php_version
+    to_number(php_version) != 7.4
```

Comment:
As mentioned in the CIS Microsoft Azure Foundations Benchmark v1.4.0 (item 9.6), the PHP version should be the latest available stable release that is PHP 8.0

```suggestion
    to_number(php_version) != 8.0
```